client/extensions/woocommerce/state/ui/shipping/zones/methods/utils.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export const getMethodName = ( methodType ) => {
```

Comment:
We have this information fetched from the server in `state/shipping-methods`, I believe we should use that instead.